srcpkgs/qperf/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'qperf'
 pkgname=qperf
 version=0.4.11
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="automake"
+makedepends="rdma-core-devel libibmad-devel"
+depends="rdma-core libibmad"
```

Comment:
stop doing this and read about 'Runtime dependencies' on the manual

Reply:
Fixed.